code/controllers/subsystem/economy.dm

```diff
@@ -104,3 +112,18 @@ SUBSYSTEM_DEF(economy)
 		return 1
 	inflation_value = max(round(((station_total / bank_accounts_by_id.len) / station_target), 0.1), 1.0)
 	return inflation_value
+
+///
+/**
+ * Proc for adding mail depending on living humans.
+ *
+ * Called by economy fired, though you could call this manually on the SS to add incoming mail with sanity.
+ **/
+/datum/controller/subsystem/economy/proc/generate_mail()
```

Comment:
is it necessary to loop through every player just to see if you can send mail every economy tick?

Reply:
to be fair econ ticks something like once every 5 minutes